05_stringer/patrickmarabeas/main_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := "127.0.0.1"
+	got := buf.String()
+
+	t.Run("Main function", func(t *testing.T) {
```

Comment:
I don't think there is much benefit to creating a subtest under this test function as the main function will only ever have the one output. It is a lot simpler if you just do a straight assertion, rather than running another subtest and doing the assertion in there. The only difference will be how the output of your test looks when running it.

```go
// Running as a subtest
--- PASS: TestMainOutput (0.08s)
    --- PASS: TestMainOutput/MainFunction (0.08s)

// Running as a stand alone test
--- PASS: TestMainOutput (0.08s)
```

Reply:
Resolved